src/beeware_docs_tools/build_po_translations.py

```diff
@@ -44,15 +44,37 @@ def main():
         final_dir = Path(final_dir)
 
         for language in args.language_code:
-            print(f"Processing {language}")
+            print(f"Processing primary {language} content")
             merge_translation_files(
                 source_dir=SOURCE_DIR / language / "LC_MESSAGES",
                 template_dir=SOURCE_DIR / "templates",
                 destination_dir=final_dir / language / "LC_MESSAGES",
             )
 
             shutil.copytree(
-                final_dir / language, SOURCE_DIR / language, dirs_exist_ok=True
+                final_dir / language,
+                SOURCE_DIR / language,
+                dirs_exist_ok=True,
+            )
+
+            print(f"Processing shared {language} content")
+            merge_translation_files(
+                source_dir=Path(__file__).parent
+                / "shared_content"
+                / "locales"
+                / language
+                / "LC_MESSAGES",
+                template_dir=Path(__file__).parent
+                / "shared_content"
+                / "locales"
+                / "templates",
+                destination_dir=final_dir / "shared_content" / language / "LC_MESSAGES",
+            )
+
+            shutil.copytree(
+                final_dir / "shared_content",
+                Path(__file__).parent / "shared_content" / "locales",
+                dirs_exist_ok=True,
```

Comment:
As with the PO file - if we're going to do the same thing twice in a row with different inputs - can we structure the code like that?